waspc/e2e-test/Tests/WaspBuildTest.hs

```diff
@@ -0,0 +1,15 @@
+module Tests.WaspBuildTest (waspBuild) where
```

Comment:
Btw you don't need to suffix files with Test if you don't want to, we are doing that in unit tests because then I can give regex to Tasty which it uses to pick up tests, but you don't need that here. If you like it thought, I am ok with it, just mentioning.